static/register.html

```diff
@@ -141,8 +289,6 @@
     </div>
 </div>
 
-<script type="text/javascript" src="/min/modernizr-min.js"></script>
-<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
y

Reply:
moved to <head> so jQuery AJAX can be used in functions declared up there.